TelemetrySite/client/src/contextForm/ContextForm.jsx

```diff
@@ -0,0 +1,453 @@
+/**
+ * EHughes
+ * 2024
```

Comment:
I'm not huge on putting author names and dates in EVT files because it gets stale fast, and a lot of people end up touching a lot of files. I don't feel that strongly about it though, so it's up to you whether you keep it